sql/moz-fx-data-shared-prod/telemetry_derived/agg_active_users_v1/init.sql

```diff
@@ -0,0 +1,61 @@
+CREATE OR REPLACE TABLE
+  `mozilla-public-data`.telemetry_derived.agg_active_users_v1
+PARTITION BY
+  DATE(submission_date)
+AS
+SELECT
+  activity_segment AS segment,
+  app_version AS app_version,
+  attribution_campaign,
+  attribution_content,
+  attribution_experiment,
+  attribution_medium,
+  attribution_source,
+  attribution_variation,
+  city,
+  country,
+  device_model,
+  distribution_id,
+  is_default_browser,
+  locale,
+  normalized_app_name AS app_name,
+  normalized_channel AS channel,
+  normalized_os AS os,
+  normalized_os_version AS os_version,
+  os_version_major,
+  os_version_minor,
+  submission_date,
+  COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS dau,
+  COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS wau,
+  COUNT(DISTINCT client_id) AS mau,
+  COUNT(DISTINCT IF(is_new_profile IS TRUE, client_id, NULL)) AS new_profiles,
+  SUM(ad_click) AS ad_clicks,
+  SUM(organic_search_count) AS organic_search_count,
+  SUM(search_count) AS search_count,
+  SUM(search_with_ads) AS search_with_ads
+FROM
+  `moz-fx-data-shared-prod.telemetry_derived.unified_metrics_v1`
+WHERE
+  submission_date = @submission_date
```

Comment:
```suggestion
  submission_date = DATE(@submission_date)
```

Just to be safe I'd maybe cast the @submission_date param as DATE.

Reply:
Also, this is an `init.sql` and will be run to initialize the table. You could just say `submission_date < DATE("2022-05-11")`

This way it'll grab all the data until submission date from the source table. There won't be a need to create any additional backfills, the dag should be able to just take over from here.